src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -188,17 +213,68 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Sch
         }
 
         public OnnxTransform(IHostEnvironment env, string modelFile)
-            : this(env, new Arguments() { ModelFile = modelFile, InputColumns = new string[] { }, OutputColumns = new string[] { } })
+            : this(env, modelFile, -1, false)
+        {
+
+        }
+
+        public OnnxTransform(IHostEnvironment env, string modelFile, int gpuDeviceId = -1, bool fallbackToCpu = false)
```

Comment:
> public OnnxTransform(IHostEnvironment env, string modelFile, int gpuDeviceId = -1, bool fallbackToCpu = false) [](start = 7, length = 111)

maybe remove the previous ctor. Calling this without providing the optional params will fail with "ambiguous call" messages.